Requirement already satisfied: geopy in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (2.4.1)
Requirement already satisfied: kagglehub[pandas-datasets] in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (0.3.10)
Requirement already satisfied: packaging in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from kagglehub[pandas-datasets]) (24.2)
Requirement already satisfied: pyyaml in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from kagglehub[pandas-datasets]) (6.0.2)
Requirement already satisfied: requests in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from kagglehub[pandas-datasets]) (2.32.3)
Requirement already satisfied: tqdm in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from kagglehub[pandas-datasets]) (4.67.1)
Requirement already satisfied: pandas in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from kagglehub[pandas-datasets]) (2.2.3)
Requirement already satisfied: geographiclib<3,>=1.52 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from geopy) (2.0)
Requirement already satisfied: numpy>=1.22.4 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from pandas->kagglehub[pandas-datasets]) (2.0.2)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from pandas->kagglehub[pandas-datasets]) (2.9.0.post0)
Requirement already satisfied: pytz>=2020.1 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from pandas->kagglehub[pandas-datasets]) (2025.1)
Requirement already satisfied: tzdata>=2022.7 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from pandas->kagglehub[pandas-datasets]) (2025.1)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from requests->kagglehub[pandas-datasets]) (3.4.1)
Requirement already satisfied: idna<4,>=2.5 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from requests->kagglehub[pandas-datasets]) (3.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from requests->kagglehub[pandas-datasets]) (2.3.0)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from requests->kagglehub[pandas-datasets]) (2025.1.31)
Requirement already satisfied: colorama in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from tqdm->kagglehub[pandas-datasets]) (0.4.6)
Requirement already satisfied: six>=1.5 in c:\users\user\appdata\local\programs\python\python39\lib\site-packages (from python-dateutil>=2.8.2->pandas->kagglehub[pandas-datasets]) (1.17.0)
